src/Sentry.Unity.Editor/SentryWindow.cs

```diff
@@ -63,85 +93,250 @@ private void CheckForAndConvertJsonConfig()
+                new GUIContent("Enable Offline Caching", ""),
+                Options.EnableOfflineCaching);
+            Options.MaxCacheItems = EditorGUILayout.IntField(
+                new GUIContent("Max Cache Items", "The maximum number of items to keep in cache. SDK " +
+                                                  "deletes the oldest and migrates the sessions to the next to " +
+                                                  "maintain the integrity of your release health stats.\nDefault: 30"),
+                Options.MaxCacheItems);
+
+            Options.InitCacheFlushTimeout = EditorGUILayout.IntField(
+                new GUIContent("Init Flush Timeout [ms]", "The timeout that limits how long the SDK " +
+                                                          "will attempt to flush existing cache during initialization."),
+                Options.InitCacheFlushTimeout);
+
+            EditorGUILayout.EndToggleGroup();
+
+            EditorGUILayout.Space();
+            EditorGUI.DrawRect(EditorGUILayout.GetControlRect(false, 1), Color.gray);
+            EditorGUILayout.Space();
+
+            // Options.RequestBodyCompressionLevel = (CompressionLevelWithAuto)EditorGUILayout.EnumPopup(
+            //     new GUIContent("Compress Payload", "The level of which to compress the Sentry event " +
+            //                                        "before sending to Sentry."),
+            //     Options.RequestBodyCompressionLevel);
+
+            var sampleRate = Options.SampleRate ??= 1.0f;
+            sampleRate = EditorGUILayout.Slider(
+                new GUIContent("Event Sample Rate", "Indicates the percentage of events that are " +
+                                                    "captured. Setting this to 0.1 captures 10% of events. " +
+                                                    "Setting this to 1.0 captures all events. Events are picked randomly."),
+                sampleRate, 0.01f, 1);
+            Options.SampleRate = (sampleRate < 1.0f) ? sampleRate : null;
+
+            Options.ShutdownTimeout = EditorGUILayout.IntField(
+                new GUIContent("Shut Down Timeout [ms]", "How many seconds to wait before shutting down to " +
+                                                         "give Sentry time to send events from the background queue."),
+                Options.ShutdownTimeout);
+
+            Options.MaxQueueItems = EditorGUILayout.IntField(
+                new GUIContent("Max Queue Items", "The maximum number of events to keep while the " +
+                                                  "worker attempts to send them."),
+                Options.MaxQueueItems
+            );
+        }
+
+        private void DisplaySessions()
+        {
+            Options.AutoSessionTracking = EditorGUILayout.BeginToggleGroup(
+                new GUIContent("Auto Session Tracking", "Whether the SDK should start a session " +
```

Comment:
A bit misleading because we can also end if the app is on the background for 30 seconds or more. If the platform supports that.